client/src/main.rs

```diff
@@ -160,9 +163,11 @@ pub async fn main() -> Result<(), Box<dyn Error>> {
         .get_matches();
 
     let _address = matches.value_of("address").unwrap();
```

Comment:
This should be renamed to `address`. Some of the variables in here were prefixed with an underscore temporarily to pass clippy, until they're used.